tests/utils.go

```diff
@@ -74,13 +78,18 @@ var KubeVirtOcPath = ""
 var KubeVirtVirtctlPath = ""
 var KubeVirtInstallNamespace string
 
+var DeployTestingInfrastructureFlag = false
+var PathToTestingInfrastrucureManifests = ""
+
 func init() {
-	flag.StringVar(&KubeVirtVersionTag, "tag", "latest", "Set the image tag or digest to use")
-	flag.StringVar(&KubeVirtRepoPrefix, "prefix", "kubevirt", "Set the repository prefix for all images")
+	flag.StringVar(&KubeVirtVersionTag, "docker-tag", "latest", "Set the image tag or digest to use")
```

Comment:
Can you rename it to `container-tag` and `container-prefix`, just for the case when we will switch to quay?